helm/kyverno-policies-observability/templates/PodMonitor.yaml

```diff
@@ -0,0 +1,46 @@
+# THIS FILE IS GENERATED WITH 'make generate' - DO NOT EDIT MANUALLY
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: configure-pod-monitor-labelling-schema
+spec:
+  rules:
+    - name: configure-pod-monitor-labelling-schema
+      match:
+        resources:
+          kinds:
+          - monitoring.coreos.com/v1/PodMonitor
+      mutate:
+        patchStrategicMerge:
+          spec:
+            podMetricsEndpoints:
+              - relabelings:
+                  # Pod Monitors are currently only supported on management clusters so we can enforce the cluster_id and cluster_type.
+                - replacement: "{{ .Values.managementCluster.name }}"
+                  targetLabel: cluster_id
+                - replacement: "management_cluster"
+                  targetLabel: cluster_type
+                - replacement: "{{ .Values.provider.kind }}"
+                  targetLabel: provider
+                - replacement: "{{ .Values.managementCluster.name }}"
+                  targetLabel: installation
+                - sourceLabels: [__meta_kubernetes_namespace]
+                  targetLabel: namespace
+                - sourceLabels: [__meta_kubernetes_pod_name]
+                  targetLabel: pod
+                - sourceLabels: [__meta_kubernetes_pod_container_name]
+                  targetLabel: container
+                - sourceLabels: [__meta_kubernetes_pod_node_name]
+                  targetLabel: node
+                - sourceLabels: [__meta_kubernetes_node_label_role]
+                  targetLabel: role
+                - replacement: "{{ .Values.managementCluster.customer }}"
+                  targetLabel: customer
+                ## We get the organization from the namespace (org-organization_name)
+                - replacement: "{{ `{{` }} request.namespace {{ `}}` }}" 
+                  targetLabel: organization
+                ## Then we remove the org prefix
+                - sourceLabels: [organization]
+                  regex: org-(.*)
```

Comment:
What is the reason to remove the `org-` prefix? I would rather leave this as is, as it introduces less confusion.